packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -68,78 +78,99 @@
                     color-scheme="positive" />
             </div>
 
+            <template v-if="isLoading">
+                <restaurant-content-skeleton
+                    :width-percentage="90"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="80"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="40"
+                    :class="[
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton :width-percentage="90" />
```

Comment:
I would probably move it out this block and the rating part to a separate component as this component is already quite busy

Reply:
I'm not sure the best way to do that and preserve the grid/flex styling of the parent